frontend/src/store/types.js

```diff
@@ -0,0 +1,25 @@
+export const mutations = {
+  UPDATE_QUESTIONS: 'updateQuestions',
+  SET_COMPUTER_ID: 'setComputerId',
+  SET_USER: 'setUser',
+  SET_PROPERTY: 'setProperty',
+}
+
+export const actions = {
+  FETCH_EVENT_ID_BY_CODE: 'fetchEventIdByCode',
+  SUBMIT_QUESTION: 'submitQuestion',
+  FETCH_EVENT: 'fetchEvent',
+  VOTE: 'vote',
+  JOIN_EVENT: 'joinEvent',
+  WITHDRAW_QUESTION: 'withdrawQuestion',
+  UPDATE_QUESTIONS: 'updateQuestions',
+  CREATE_EVENT: 'createEvent',
```

Comment:
you shouldn't mix user and event action types in this file. you probably want to define these in respective modules and export them from there.